app/templates/auth/login.html

```diff
@@ -1,47 +1,76 @@
 {% extends "base.html" %}
-{% from "auth/_formhelpers.html" import render_field %}
+<!-- {% from "auth/_formhelpers.html" import render_field %} -->
+{% from 'bootstrap5/form.html' import render_form, render_field, render_form_row %}
 
 {% block content %}
-<main class="form-login">
-    <div class="container">
-        <div class="row">
-            <div class="col-12">
-                <div class="card-wrapper mt-5">
-                    <div class="card login-card">
-                        <div class="row">
-                            <div class="col-12">
-                                <h1 class="my-3 pt-3 mt-lg-5 text-center">Login</h1>
-                            </div>
-                        </div>
+<!--start-content-->
+<section class="form-login container-fluid mt-3 mt-md-5" id="login-form">
+    <!--start-row-->
```

Comment:
I worry a little about the use of comments to denote the start and end of a block of code. It might just be me being pedantic, but we have almost as much code comments here specifying the blocks of code, than the actual code itself, and it makes the review a little difficult. I'm going to ask @stefdworschak for his opinion on this too, I do think we should consider removing some of these in due course though.

Reply:
@auxfuse Generally, I would not add comments for the commenting's sake. Comments are necessary if the developer did something uncommon or very complex for example. Otherwise, the code should be so readable that no comments are necessary to understand what is going on. Especially in a markup language like HTML this is not really necessary.

Too many comments could also be a sign of too much complexity or hard to understand code which we should simplified. So I would suggest to remove these sort of comments as they are not needed.